Vungle/src/main/java/com/mopub/mobileads/VungleAdapterConfiguration.java

```diff
@@ -42,8 +48,13 @@ public String getAdapterVersion() {
     @Nullable
     @Override
     public String getBiddingToken(@NonNull Context context) {
-        final String bidToken = Vungle.getAvailableBidTokens(10);
-        return bidToken;
+        if (initBidToken.compareAndSet(false, true)) {
+            String token = Vungle.getAvailableBidTokens(10);
+            tokenReference.set(token);
+        } else {
+            refreshBidderToken(context);
```

Comment:
so each call will get the previous bidtoken? 
Looks like it's unexpected... shouldn't we have a timeout to get the newest one before returning one from the memory?